sdk/create-dapp/README.md

```diff
@@ -0,0 +1,33 @@
+# @mysten/create-dapp
+
+`@mysten/create-dapp` is a CLI tool that helps you to create a new dapp project.
```

Comment:
Lol this is such a small thing, but what's the correct casing for a decentralized app? dApp, DApp, dapp? I'm good with whatever as long as we're consistent